### ISSUE TITLE
It is supposed to be installed, but it is not.

### ISSUE THREAD

**Issue by uga-rosa** [b4b4r07/afx#32]
## WHAT

It is indeed installed and the commands are usable, but the result of `afx init` looks like this.

```
## package "junegunn/fzf" is not installed
## package "junegunn/fzf" is not installed
```

My github.yaml
```yaml
github:
  - name: junegunn/fzf
    description: A command-line fuzzy finder
    owner: junegunn
    repo: fzf
    command:
      build:
        steps:
          - ./install --all
      link:
        - from: 'bin/fzf'
        - from: 'bin/fzf-tmux'
    plugin:
      sources:
        - ~/.fzf.zsh
      env:
        FZF_DEFAULT_COMMAND: fd --type f
        FZF_DEFAULT_OPTS: --height 50% --reverse
        FZF_CTRL_T_COMMAND: rg --files --hedden --follow --glob "!.git/*"
        FZF_CTRL_T_OPTS: --preview "bat --color=always --style=header,grid --line-range :100 {}"
        FZF_ALT_C_COMMAND: fd --type d
```

## WHY

If it's a bug, that's the report. If there is something wrong with the settings, please let me know.

**Comment by babarot** [comment]
@uga-rosa 

Thank you for reporting! It was a bug so I fixed it.

Now it has been solved in master branch or v0.1.6. Please try it out!

**Comment by uga-rosa** [comment]
Thank you for the quick fix!
